src/components/Nav/NavRight.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { withRouter, Link, useHistory } from 'react-router-dom';
+import { TOKEN_KEY } from '../../config.js';
+import styled from 'styled-components';
+
+function NavRight() {
+  let history = useHistory();
+
+  const logOut = () => {
+    localStorage.removeItem(TOKEN_KEY);
+    history.push('/');
+  };
+
+  const token = localStorage.getItem(TOKEN_KEY);
+
+  return (
+    <NavRightDes>
+      <NavRightDesUl>
+        {token
+          ? TOKENIN.map(tokenIn => {
+              const { key, content, url } = tokenIn;
+              return (
+                <li key={key}>
+                  <Link to={url}> {content}</Link>
+                </li>
+              );
+            })
+          : TOKENOUT.map(tokenout => {
+              const { key, content, url } = tokenout;
+              return (
+                <li key={key}>
+                  <Link to={url}> {content}</Link>
+                </li>
+              );
+            })}
```

Comment:
```suggestion
           const targetArray = token ? TOKENIN ? TOKENOUT;
           
           {targetArray.map(tokenIn => {
              const { key, content, url } = tokenIn;
              return (
                <li key={key}>
                  <Link to={url}> {content}</Link>
                </li>
              );
            }
```
이런식으로도 표현 가능해보입니다! 별개로 TOKENIN, TOKENOUT 같은 네이밍은 적절치 않아보이네요! 좀 더 의미가 드러나는 이름으로 변경해주세요

Reply:
넵 수정했습니다~ 감사합니다:)